dags/mapper_dag.py

```diff
@@ -0,0 +1,77 @@
+from datetime import datetime
+import json
+import sys
+
+from airflow.decorators import dag, task
+from airflow.models.param import Param
+
+from rikolti.metadata_mapper.lambda_shepherd import get_vernacular_pages, get_collection
+from rikolti.metadata_mapper.lambda_function import map_page as lambda_map_page
+
+
+@task()
+def fetch_pages(params=None):
+    if not params:
+        return False
+
+    payload = params.get('payload')
+    collection_id = payload.get('collection_id', 0)
+    # raise an error?
+    if not collection_id:
+        return []
+
+    pages = get_vernacular_pages(
+                collection_id)
```

Comment:
I went ahead and factored out the rollup reporting from `lambda_shepherd.map_collection` into `lambda_shepherd.get_mapping_summary` so that it can be called from the mapper dag. Lmk what you think.